src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -2123,5 +2123,25 @@ Class C
                 state.AssertNoCompletionSession()
             End Using
         End Sub
+
+        <WorkItem(1659, "https://github.com/dotnet/roslyn/issues/1659")>
+<Fact, Trait(Traits.Feature, Traits.Features.Completion)>
```

Comment:
Are you trying to prove a point to @mattwar by having an un-indented attribute in VB after copy/pasting a method? :wink: